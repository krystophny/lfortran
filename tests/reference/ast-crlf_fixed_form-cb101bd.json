{
    "basename": "ast-crlf_fixed_form-cb101bd",
    "cmd": "lfortran --fixed-form --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/crlf_fixed_form.f",
    "infile_hash": "ed8f4cb1370f9d377b39bdb0f16053e9c86bd0d6fa2c5b97c304a7a3",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "ast-crlf_fixed_form-cb101bd.stderr",
    "stderr_hash": "0763eb8c7a074f538f7f447551bd89572706fd8650a0b57450f924c5",
    "returncode": 127
}