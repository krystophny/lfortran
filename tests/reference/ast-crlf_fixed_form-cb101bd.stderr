/bin/sh: line 1: lfortran: command not found
